.github/workflows/static-checks.yml

```
@@ -141,5 +141,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          args: "--exclude-pattern '.*dependabot\[bot\]@users.noreply.github.com'"
+        args: '--exclude-pattern ".*dependabot\[bot\]@users\.noreply\.github\.com"'
```

- You are passing double quotes inside single quotes. This means the shell/command will treat the regex as quoted literally, and the regex engine may not parse it correctly. Might make inner double quotes unnecessary
- The backslashes are correct for escaping, but the extra inner quotes are not.

```suggestion
        args: --exclude-pattern '.*dependabot\[bot\]@users\.noreply\.github\.com'
```